collect_app/src/main/java/org/odk/collect/android/storage/StoragePathProvider.java

```diff
@@ -96,10 +99,6 @@ public String getFormDbPath(String filePath) {
         return getDbPath(getDirPath(StorageSubdirectory.FORMS), filePath);
     }
 
-    public String getAbsoluteFormFilePath(String filePath) {
```

Comment:
It was clear when working with `ServerFormDownloader` that this part of the interface feels like too much as the provider becomes responsible for calculations that are based on it's state but have logic (calculating a relative or absolute path) that can live separately. All the extra helpers increase the coupling on whatever object is using it.